ionoscloud/data_source_dbaas_redisdb_replicaset.go

```diff
@@ -0,0 +1,208 @@
+							Computed:    true,
+						},
+						"lan_id": {
+							Type:        schema.TypeString,
+							Description: "The numeric LAN ID to connect your Replicaset to.",
+							Computed:    true,
+						},
+					},
+				},
+			},
+			"credentials": {
+				Type:        schema.TypeList,
+				Description: "The credentials for your Replicaset.",
+				Computed:    true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"password": {
+							Type:        schema.TypeString,
+							Description: "The password for your Replicaset.",
+							Computed:    true,
+						},
+						"username": {
+							Type:        schema.TypeString,
+							Description: "The username for your Replicaset.",
+							Computed:    true,
+						},
+					},
+				},
+			},
+			"eviction_policy": {
+				Type:        schema.TypeString,
+				Description: "The eviction policy of the Redis Replicaset.",
+				Computed:    true,
+			},
+			"maintenance_window": {
+				Type:        schema.TypeList,
+				Description: "A weekly 4 hour-long window, during which maintenance might occur.",
+				Computed:    true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"time": {
+							Type:        schema.TypeString,
+							Description: "Start of the maintenance window in UTC time.",
+							Computed:    true,
+						},
+						"day_of_the_week": {
+							Type:        schema.TypeString,
+							Description: "The name of the week day.",
+							Computed:    true,
+						},
+					},
+				},
+			},
+			"persistence_mode": {
+				Type:        schema.TypeString,
+				Description: "The persistence mode of the Redis Replicaset.",
+				Computed:    true,
+			},
+			"redis_version": {
+				Type:        schema.TypeString,
+				Description: "The version of Redis used in the Replicaset.",
+				Computed:    true,
+			},
+			"replicas": {
+				Type:        schema.TypeInt,
+				Description: "The number of replicas in the Replicaset.",
+				Computed:    true,
+			},
+			"resources": {
+				Type:        schema.TypeList,
+				Description: "The resources allocated to the Replicaset.",
+				Computed:    true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"cores": {
+							Type:        schema.TypeInt,
+							Description: "The number of CPU cores per instance.",
+							Computed:    true,
+						},
+						"ram": {
+							Type:        schema.TypeInt,
+							Description: "The amount of memory per instance in gigabytes (GB).",
+							Computed:    true,
+						},
+						"storage": {
+							Type:        schema.TypeInt,
+							Description: "The amount of storage per instance in gigabytes (GB).",
+							Computed:    true,
+						},
+					},
+				},
+			},
+		},
+	}
+}
+
+func dataSourceDBaaSRedisDBReplicaSetRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	client := meta.(services.SdkBundle).RedisDBClient
+	id, idOk := d.GetOk("id")
+	name, nameOk := d.GetOk("display_name")
```

Comment:
@avirtopeanu-ionos please change `name` to `displayName`, even though it's the same thing, let's name the var as the attribute.

Reply:
since tihs is a nitpick, can we do this in the main branch?